test/app_engine_test_utils.py

```diff
@@ -0,0 +1,25 @@
+from google.appengine.ext import testbed
+from google.appengine.ext import ndb
+
+
+def activate_app_engine_testbed():
```

Comment:
this looks like it's doing the same thing as the nose2-gae init.

https://github.com/udacity/nose2-gae/blob/master/nose2gae/__init__.py